source/Octopus.Tentacle.Tests/Kubernetes/KubernetesDirectoryInformationProviderFixture.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using FluentAssertions;
+using NSubstitute;
+using NSubstitute.Extensions;
+using NUnit.Framework;
+using Octopus.Diagnostics;
+using Octopus.Tentacle.Kubernetes;
+using Octopus.Tentacle.Util;
+
+namespace Octopus.Tentacle.Tests.Kubernetes
+{
+    public class KubernetesDirectoryInformationProviderFixture
+    {
+        // Sizes
+        const ulong Megabyte = 1000 * 1000;
+        const ulong Mebibyte = 1024 * 1024;
+        const ulong Gibibyte = 1024 * 1024 * 1024;
+        
+        [SetUp]
+        public void SetUp()
+        {
+            Environment.SetEnvironmentVariable(KubernetesConfig.NamespaceVariableName, "test-namespace");
```

Comment:
Do we need this anymore?

Reply:
Ah you're right! Since we just DI it in, we don't :)